lib/blocks.js

```diff
@@ -949,6 +973,27 @@ var findNextNonspace = function() {
     this.indented = this.indent >= CODE_INDENT;
 };
 
+function measureNonspaceColumn(line) {
+    // This code is copied from findNextNonspace above
+    var i = 0;
+    var cols = 0;
+    var c;
+
+    while ((c = line.charAt(i)) !== "") {
+        if (c === " ") {
+            i++;
```

Comment:
nit to the 10th degree: i++ could be under the if block so won't repeat twice?

Reply:
Yeah, it could, but I'm going to leave this as-is since it's copied from the non-forked function above